### PR TITLE
Fix to #136

### PR DESCRIPTION
**Pull request by msimmon6**
Checks for comment before creating the single indent width.

### CODE REVIEW

**Comment by Algomorph** [comment]
@msimmon6 , wouldn't this only work when the multi-line comment line starts with "*" (which it by no means is required to)? I think we discussed the solution that would work in all cases (i.e. detect the start & end of a comment block, if the comment block doesn't end on the same line, keep track of this by setting & keeping status as "in multi-line comment" or the like, and when you detect the end of the comment block, resume normal processing -- taking the length of the final comment line until the "*/" into account when computing the indent for that line).

**Review by Algomorph — CHANGES_REQUESTED** [review]
(See earlier comment)

**Comment by Algomorph** [comment]
Just following up on this, @msimmon6. When do you think you'll have a chance to address the changes?

**Review by Algomorph — CHANGES_REQUESTED** [review]
[EDITED] Matt, I'm going to let the merges slide this time, but I had to try and figure out what your changes were from individual commits because of the merges. The review feature cannot seem to isolate your specific changes when you do merges from master, so please do `rebase -i` next time.

Please tell me about the `beautify issue` below, and, once we come to a decision on that, I can merge. I think the commit history might still be okay after this...

**Comment by Algomorph** [comment]
Please, always remember to pull master, check out your branch again, run `git rebase -i master` (& fix conflicts, if any), and re-push your branch any time there have been other changes applied to master before you want to merge a PR.

File: options_compiled.js [hunk 1]
```diff
@@ -5975,10 +5975,7 @@ function saveOptions() {
                     status.textContent = '';
                 }, 750);
             });
-        document.getElementById('optionsTextArea').value = beautify(optionsStr, {
-            indent_size: 4,
-            space_in_empty_paren: true
-        });
+
```
Comment (Algomorph) [inline options_compiled.js:5978]: Why are we getting rid of "beautify" here?

Comment (msimmon6) [reply options_compiled.js:5978]: We shouldn't, that was a mistake - woops!

**Comment by msimmon6** [comment]
I know I pulled and rebased this time, I probably forgot to the first time I I made this PR. I'll be sure to do it from here on out, sorry about the trouble.

**Review by Algomorph — APPROVED** [review]
Looks good, thanks!

**Pull request merged**